source/_integrations/ping.markdown

```diff
@@ -63,10 +63,6 @@ The integration exposes the different round trip times milliseconds as entities:
 
 **These entities are disabled by default and can be enabled in the UI if needed.**
 
-{% note %}
-When run on Windows systems, the round trip time attributes are rounded to the nearest millisecond and the mdev value is unavailable.
```

Comment:
could it be about pinging windows machines?

Reply:
I don't think it applies since the timing of the roundtrip is mostly kept on the host system. Let's await the code owner.